samples/snippets/src/main/java/com/example/spanner/UpdateBackupScheduleSample.java

```diff
@@ -0,0 +1,81 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+// [START spanner_update_backup_schedule]
+
+import com.google.cloud.spanner.admin.database.v1.DatabaseAdminClient;
+import com.google.protobuf.Duration;
+import com.google.protobuf.FieldMask;
+import com.google.spanner.admin.database.v1.BackupSchedule;
+import com.google.spanner.admin.database.v1.BackupScheduleName;
+import com.google.spanner.admin.database.v1.BackupScheduleSpec;
+import com.google.spanner.admin.database.v1.CreateBackupEncryptionConfig;
+import com.google.spanner.admin.database.v1.CrontabSpec;
+import com.google.spanner.admin.database.v1.UpdateBackupScheduleRequest;
+import java.io.IOException;
+
+class UpdateBackupScheduleSample {
+
+  static void updateBackupSchedule() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project";
+    String instanceId = "my-instance";
+    String databaseId = "my-database";
+    String backupScheduleId = "my-backup-schedule";
+    updateBackupSchedule(projectId, instanceId, databaseId, backupScheduleId);
+  }
+
+  static void updateBackupSchedule(
+      String projectId, String instanceId, String databaseId, String backupScheduleId)
+      throws IOException {
+    BackupScheduleName backupScheduleName =
+        BackupScheduleName.of(projectId, instanceId, databaseId, backupScheduleId);
+    final CreateBackupEncryptionConfig encryptionConfig =
+        CreateBackupEncryptionConfig.newBuilder()
+            .setEncryptionType(CreateBackupEncryptionConfig.EncryptionType.USE_DATABASE_ENCRYPTION)
+            .build();
+    final BackupSchedule backupSchedule =
+        BackupSchedule.newBuilder()
+            .setName(backupScheduleName.toString())
+            .setRetentionDuration(Duration.newBuilder().setSeconds(3600 * 48))
+            .setSpec(
+                BackupScheduleSpec.newBuilder()
+                    .setCronSpec(CrontabSpec.newBuilder().setText("45 15 * * *").build())
+                    .build())
+            .setEncryptionConfig(encryptionConfig)
+            .build();
+
+    try (DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.create()) {
+      final FieldMask fieldMask =
+          FieldMask.newBuilder()
+              .addPaths("retention_duration")
+              .addPaths("spec.cron_spec.text")
+              .addPaths("encryption_config")
+              .build();
+      final BackupSchedule updatedBackupSchedule =
+          databaseAdminClient.updateBackupSchedule(
+              UpdateBackupScheduleRequest.newBuilder()
+                  .setBackupSchedule(backupSchedule)
+                  .setUpdateMask(fieldMask)
+                  .build());
+      System.out.println(
+          String.format("Updated backup schedule: %s", updatedBackupSchedule.getName()));
```

Comment:
```suggestion
          String.format("Updated backup schedule: %s", updatedBackupSchedule.toString()));
```

Reply:
The updated log does not exactly follow the suggestion but uses a middle ground to avoid major refactoring.